client/components/tile-grid/README.md

```diff
@@ -0,0 +1,50 @@
+TileGrid (jsx)
+==========
+
+Component used to declare an area for displaying Tiles — it's the main wrapper that takes care of tile alignment and positioning.
+
+#### Props
+
+* `className`: Add your own class to the grid.
+* `isHidden`: Whether the entire grid should be hidden. Default is `false`.
+
+----------
+
+Tile (jsx)
+==========
+
+Component used to display a clickable tile with an image, call to action and description.
+
+#### Props
+
+* `buttonClassName`: Add your own class to the tile button.
+* `buttonLabel`: Text of the button.
+* `className`: Add your own class to the tile.
+* `description`: Description text.
+* `href`: URL that the item leads to upon click.
+* `image`: URL of the image.
+* `onClick`: Function, executed when the user clicks the tile,
```

Comment:
Nitpick: Period after `tile` (not comma).

Reply:
Oh well - f193f1e6c962b86e583381b28117bddde46a9b39.